hosted_control_planes/hcp-prepare/hcp-requirements.adoc

```diff
@@ -12,6 +12,8 @@ The management cluster and worker nodes must run on the same type of infrastruct
 
 The control plane is associated with a hosted cluster and runs as pods in a single namespace. When the cluster service consumer creates a hosted cluster, it creates a worker node that is independent of the control plane.
 
+If you use a multi-architecture deployment, review the latency requirements in the support matrix. In bare metal host (BMH) deployments, where the Bare Metal Operator starts machines, baseboard management controllers (BMCs) must be reachable from the hosted control plane, which can create a security concern. To avoid that issue, use BYO ISO.
```

Comment:
Rather than `To avoid that issue, use BYO ISO` I think we could word it like so:

If having the Cluster BareMetal Operator access the network where the bare-metal hosts have their BMCs in order to have Redfish automation is incompatible with your security profile, you can still use Hosted Control Planes thanks to the BYO ISO support. In BYO mode, however, OpenShift won't be able to automate the powering on of the bare-metal hosts for you